app/workflows/lists/conversion/sections/after-opening.yml

```diff
@@ -37,7 +36,7 @@ tasks:
       - title: Send relevant documents to the funding team
         guidance_summary: Help sending documents
         guidance_text: |
```

Comment:
Another one

Reply:
I think this one needs to stay on the guidance text as it has markdown for a link in it.

Reply:
This actually doesn't need the pipe. We only need it for newlines